BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/XFintoPUEO_Symmetric.py

```diff
@@ -0,0 +1,133 @@
+'''
+freqVals = [
+    83.33e6, 100.00e6, 116.67e6, 133.33e6, 150.00e6, 166.67e6, 183.34e6,
+    200.00e6, 216.67e6, 233.34e6, 250.00e6, 266.67e6, 283.34e6, 300.00e6, 
+    316.67e6, 333.34e6, 350.00e6, 366.67e6, 383.34e6, 400.01e6, 416.67e6,
+    433.34e6, 450.01e6, 466.67e6, 483.34e6, 500.01e6, 516.68e6, 533.34e6,
+    550.01e6, 566.68e6, 583.34e6, 600.01e6, 616.68e6, 633.34e6, 650.01e6,
+    666.68e6, 683.35e6, 700.01e6, 716.68e6, 733.35e6, 750.01e6, 766.68e6,
+    783.35e6, 800.01e6, 816.68e6, 833.35e6, 850.02e6, 866.68e6,883.35e6,
+    900.02e6, 916.68e6, 933.35e6, 950.02e6, 966.68e6, 983.35e6, 1000.00e6,
+    1016.70e6, 1033.40e6, 1050.00e6, 1066.70e6
+    ]
+numFreq = 60
+'''
+
+freqList = [f'{val}' for val in freqVals] # This is in the format that pueoSim wants
+big_freq_list = freqList * 6 # This is the same thing, just 6 times :) (this is to get the gain at different angles at each frequency, we have 6 angles)
+
+# Adds arguments when running code
+parser = argparse.ArgumentParser()
+parser.add_argument("NPOP", help="How many antennas are being simulated each generation?", type=int)
+parser.add_argument("WorkingDir", help="Evolutionary_Loop directory", type=Path)
+parser.add_argument("RunName", help="Run Name directory where everything will be stored", type=str)
+parser.add_argument("gen", help="The generation the loop is on", type=int)
+parser.add_argument("indiv", help="The individual in the population", type=int)
+g = parser.parse_args()
+
+# Function to read in files
+def readFile(indiv, freqNum, col):
+    uanname = Path(f'{g.WorkingDir}/Run_Outputs/{g.RunName}/{g.gen}_{indiv}_{freqNum}.uan')
+    #uanname = Path(f'{g.WorkingDir}/{g.RunName}/uan_files/{g.gen}_uan_files/{indiv}/{g.gen}_{indiv}_{freqNum}.uan')
+    #pulling data from the uan file                                                                                         
+    data = np.genfromtxt(uanname, unpack=True, skip_header=18).tolist()
+
+    return (data[col]) #returns Gain(Theta) column THIS WILL NEED TO BE CHANGED FOR WHEN WE HAVE WORKING XF OUTPUTS (maybe make column input variable for function?)                                                                               
+## Make a function to create a list for each of the columns (V and H gains)
+## This function reads in the gain at each phi for the desired thetas
+def getGains(indiv):
+	vpolGains = []
+	hpolGains = []
+
+	for freq in range(numFreq):
+		vpolGains.append(readFile(indiv, freq+1, 2))
+		hpolGains.append(readFile(indiv, freq+1, 3))
+
+	return vpolGains, hpolGains
+
+## This function writes the data to a file
+def writeGains(data, freqList, fileName):
+	with open(fileName, "a") as f:
+		for i in range(numFreq):
+			f.write(str(freqList[i]) + " " + str(data[i]) + "\n")
+	return 0
+
+
+## Here's the path to output to
+fileHeader = g.WorkingDir / 'Test_Outputs' 
+## Loop over the number of individuals
+for indiv in range(1, g.NPOP+1):
+		## Let's call the functions
```

Comment:
This sections uses tabs instead of spaces, and also has 2 levels of indentation instead of 1.  Tabs should never be used in Python (always 4 spaces for level of indentation).

Reply:
Good catch! I think my editor was defaulting to tabs--I'm surprised it didn't yell at me for inconsistency. But yes, I'll go through a replace our tabs with 4 spaces.